tests/flows/erc20_source_erc20_destination_send_and_call.go

```diff
@@ -0,0 +1,245 @@
+package flows
+
+import (
+	"context"
+	"math/big"
+
+	"github.com/ava-labs/subnet-evm/accounts/abi/bind"
+	erc20destination "github.com/ava-labs/teleporter-token-bridge/abi-bindings/go/ERC20Destination"
+	erc20source "github.com/ava-labs/teleporter-token-bridge/abi-bindings/go/ERC20Source"
+	"github.com/ava-labs/teleporter-token-bridge/tests/utils"
+	"github.com/ava-labs/teleporter/tests/interfaces"
+	teleporterUtils "github.com/ava-labs/teleporter/tests/utils"
+	"github.com/ethereum/go-ethereum/crypto"
+	. "github.com/onsi/gomega"
+)
+
+/**
+ * Deploy an ERC20 token source on the primary network
+ * Deploys ERC20Destination to Subnet A
+ * Bridges C-Chain example ERC20 tokens to Subnet A
+ * Bridge tokens from Subnet A to C-Chain using Send and Call
```

Comment:
```suggestion
 * Bridges C-Chain example ERC20 tokens to Subnet A and calls contract on Subnet A using sendAndCall
 * Bridges C-Chain example ERC20 to EOA on Subnet A, and then bridge tokens from Subnet A back
 * C-Chain and calls contract on the C-Chain using sendAndCall
```

I think this is the true flow right?